examples/destructuring-records/destructuring_records.description

```diff
@@ -1,3 +1,3 @@
-// Destructuring records is particularly useful with `query expressions`, but works anywhere you can have `var`.
+// Destructuring records is particularly useful with `query expressions` but works anywhere you can have `var`.
```

Comment:
```suggestion
// Destructuring records is particularly useful with query expressions but works anywhere you can have `var`.
```
Since "query expressions" is not code